modular_citadel/code/modules/arousal/toys/dildos.dm

```diff
@@ -138,16 +138,15 @@ obj/item/dildo/custom
 
 // Suicide acts, by request
 
-var/is_knotted = (dildo_shape == knotted) ? "They swallowed the knot" : "Their face is turning blue"
-
 /obj/item/dildo/proc/manual_suicide(mob/living/user)
 		user.visible_message("<span class='suicide'>[user] finally finishes deepthroating the [src], and their life.</span>")
 		user.adjustOxyLoss(200)
 		user.death(0)
 
 /obj/item/dildo/suicide_act(mob/living/user)
+//	is_knotted = ((src.dildo_shape == "knotted")?"They swallowed the knot":"Their face is turning blue")
```

Comment:
Yea, Reason why you did got errors previously was because a var operation like this needs to be contained within the proc.
So, now that you know this, you can uncomment this code, add the var/ prefix, and readd it to the flavour text if you want.
Otherwise remove this comment.

Reply:
That was what I tried and it gave me issues, such as the triggering character not actually dying. It displays the message correctly but it seems to break everything else. I'm just going to try to committ this, for now, as it is. People can use their imagination if they want to choke on dog-dick, though I appreciate this all for the exercise in coding this was.